src/content/basics/nodepools/index.md

```diff
@@ -0,0 +1,142 @@
+
+Assuming that the node pool ID is `a1b2c`, your `nodeSelector` should look like this (for example):
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+  nodeSelector:
+    giantswarm.io/machine-deployment: a1b2c
+```
+
+## Node pool deletion
+
+You can delete a node pool at any time using the Giant Swarm API and user interfaces. When a node pool gets deleted,
+
+- nodes in the pool will be marked as unschedulable and then drained, resulting in Pods being unassigned from the nodes
+and containers being stopped.
+- Then the actual nodes (EC2 instanced) will be removed.
+
+If you are deleting a node pool running critical workloads, we recommend to take some
+precautions:
+
+- Make sure there is at least one node pool providing enough nodes to pick
+up the workloads.
+- Double-check any taints and node selectors of your workloads to make sure they can land on different nodes.
+- For most control, cordon all of the node pool's nodes and then drain them manually, one by one.
+
+Then pay close attention to the workloads being rescheduled on other nodes once nodes are drained.
+
+## Node pools and the Giant Swarm API
+
+Handling clusters with node pools requires an API schema different from the one used for clusters
+with homogeneous worker nodes. We introduced a new API version path `v5` for this reason.
+
+Using the v5 API endpoints, you can
+
+- Create a new cluster
+- Add a node pool to a cluster
+- Rename a cluster
+- Modify a node pool (for renaming or changing scaling settings)
+- Delete a node pool
+
+## Node pools and the cluster definition YAML format
+
+Just as the Giant Swarm API schema for v4 (without node pools) and v5 (with node pools) clusters are different, the [cluster definition format](/reference/cluster-definition/) is different for both versions.
+
+The new definition schema for v5 allows for defining cluster and node pool details in one file,
+to be submitted for creation via the [`gsctl create cluster`](/reference/gsctl/create-cluster/) command.
+
+## Limitations
+
+- A node pool can have a maximum of 99 worker nodes.
+
+- At times, the EC2 instance type required by a node pool can be unavailable in certain availability zones. This can result in node pools providing less than the desired number of nodes. The more availability zones a node pool spans, the less likely this problem is to occur.
+
+- By default, clusters can have up to 5 node pools. This is limited by the AWS service quota named "IPv4 CIDR blocks per VPC" in the [VPC section](https://docs.aws.amazon.com/vpc/latest/userguide/amazon-vpc-limits.html). As the AWS account owner, you can request an increase of this limit via the AWS console.
+
+- Node pools can span a maximum of 4 different availability zones. This limit affects both the number of availability zones a single node pool can cover, as well as the number of different availability zones all node pools of a cluster can cover.
```

Comment:
We need to make it clearer that AZs selected once cannot be undone. You have 4 AZ slots and once filled they are permanent for a cluster.

Reply:
Very good catch! Updated.